syncVersion.js

```diff
@@ -0,0 +1,11 @@
+const fs = require(`fs`);
+const { version } = require(`./package.json`);
+
+const coreFile = `./src/core.js`;
+
+
+const replacedCore = fs
+    .readFileSync(coreFile, `utf8`)
+    .replaceAll(/\d+\.\d+\.5454545454-build-number/g, version);
```

Comment:
If this regex fails to match for some reason (like maybe the placeholder gets changed or the version gets moved to another file) then the version will silently remain out of sync. It would be good to detect that and fail here.